### PR TITLE
change browser.element for browser.elements in docs

### PR DESCRIPTION
**Pull request by woolter**
## Proposed changes

[//]: # change browser.element for browser.findElement in docs

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: docs/PageObjects.md [hunk 1]
```diff
@@ -35,10 +35,10 @@ import Page from './page';
 
 class LoginPage extends Page {
 
-    get username() { return browser.element('#username'); }
-    get password() { return browser.element('#password'); }
-    get submitBtn() { return browser.element('form button[type="submit"]'); }
-    get flash() { return browser.element('#flash'); }
+    get username() { return browser.findElement('#username'); }
```
Comment (christian-bromann) [inline docs/PageObjects.md]: Let's use `$` here as much as possible

File: docs/Timeouts.md [hunk 2]
```diff
@@ -70,9 +70,9 @@ Also the testing framework you use with WebdriverIO has to deal with timeouts es
 it('should login into the application', function () {
     browser.url('/login');
 
-    var form = browser.findElement('form');
-    var username = browser.findElement('#username');
-    var password = browser.findElement('#password');
+    var form = browser.$('form');
+    var username = browser.$('#username');
+    var password = browser.$('#password');
```
Comment (christian-bromann) [inline docs/Timeouts.md]: I would say, let's use single quotes consistently

File: docs/Timeouts.md [hunk 3]
```diff
@@ -70,9 +70,9 @@ Also the testing framework you use with WebdriverIO has to deal with timeouts es
 it('should login into the application', function () {
     browser.url('/login');
 
-    var form = browser.element('form');
-    var username = browser.element('#username');
-    var password = browser.element('#password');
+    var form = browser.$('form');
+    var username = browser.$('#username');
+    var password = browser.$('#password');
```
Comment (christian-bromann) [inline docs/Timeouts.md]: I would say, let's use single quotes consistently

File: docs/PageObjects.md [hunk 4]
```diff
@@ -35,10 +35,10 @@ import Page from './page';
 
 class LoginPage extends Page {
 
-    get username() { return browser.element('#username'); }
-    get password() { return browser.element('#password'); }
-    get submitBtn() { return browser.element('form button[type="submit"]'); }
-    get flash() { return browser.element('#flash'); }
+    get username() { return browser.$('#username'); }
```
Comment (christian-bromann) [inline docs/PageObjects.md]: just `$` instead of `browser.$` since it is a global

**Comment by christian-bromann** [comment]
Also would you mind rebasing on top of master?
```sh
$ git checkout master
$ git pull origin master --ff-only
$ git checkout changeElementForFindElement
$ git rebase master
$ git push origin changeElementForFindElement -f
```

**Comment by woolter** [comment]
`
16:32:16 in webdriverio/docs on  changeElementForFindElement [?] took 4s
➜ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.

16:33:20 in webdriverio/docs on  master [?]
➜ git pull origin master --ff-only
From github.com:woolter/webdriverio
 * branch              master     -> FETCH_HEAD
Already up to date.

16:33:29 in webdriverio/docs on  master [?] took 2s
➜ git checkout changeElementForFindElement
Switched to branch 'changeElementForFindElement'

16:33:39 in webdriverio/docs on  changeElementForFindElement [?]
➜ git rebase master
Current branch changeElementForFindElement is up to date.

16:33:45 in webdriverio/docs on  changeElementForFindElement [?]
➜ git push origin changeElementForFindElement -f
Counting objects: 25, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (25/25), done.
Writing objects: 100% (25/25), 2.70 KiB | 460.00 KiB/s, done.
Total 25 (delta 21), reused 0 (delta 0)
remote: Resolving deltas: 100% (21/21), completed with 7 local objects.
To github.com:woolter/webdriverio.git
 + ad404b30...fa75dc75 changeElementForFindElement -> changeElementForFindElement (forced update)

16:34:29 in webdriverio/docs on  changeElementForFindElement [?] took 4s
`

File: docs/Multiremote.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ import { multiremote } from 'webdriverio';
     await browser.url('http://json.org');
 
     // click on an element at the same time
-    const elem = await browser.$('#someElem');
+    const elem = await $('#someElem');
```
Comment (christian-bromann) [inline docs/Multiremote.md]: Actually this example requires `browser.$` because it is not run with the wdio testrunner.

File: docs/SetupTypes.md [hunk 6]
```diff
@@ -22,10 +22,10 @@ const { remote } = require('webdriverio');
 
     await browser.url('https://duckduckgo.com/')
 
-    const inputElem = await browser.$('#search_form_input_homepage')
+    const inputElem = await $('#search_form_input_homepage')
```
Comment (christian-bromann) [inline docs/SetupTypes.md]: same here

File: docs/SetupTypes.md [hunk 7]
```diff
@@ -22,10 +22,10 @@ const { remote } = require('webdriverio');
...
     await inputElem.setValue('WebdriverIO')
 
-    const submitBtn = await browser.$('#search_button_homepage')
+    const submitBtn = await $('#search_button_homepage')
```
Comment (christian-bromann) [inline docs/SetupTypes.md]: same here

File: docs/SetupTypes.md (hunk 6 above)
Comment (woolter) [reply docs/SetupTypes.md]: fixed

File: docs/SetupTypes.md (hunk 7 above)
Comment (woolter) [reply docs/SetupTypes.md]: fixed

**Pull request merged**